### DISCUSSION TITLE
Adding labelling to issues

### DISCUSSION THREAD

**Discussion by cuihtlauac** [ocaml/ocaml.org#674] [Ideas]
Here are three rules I propose:

1. No issue w/o label. An issue which does not bear any label is an issue which hasn't been reviewed by anybody from the team yet. 
1. use of the upstream label: should always have a reference to an upstream issue
1. use of the "revisit" label: team needs to make a decision on the item

**Comment by tmattio** [comment]
This sounds very good to me.

For the 3rd point, it might be worth seeing if we prefer to track these in [the board](https://github.com/orgs/ocaml/projects/2/views/6) instead
But in any case, I don't have a concern with any of these so I suggest that we try and see how it goes 🙂

**Comment by cuihtlauac** [comment]
I suggest we close this one, we're doing it now
